etc/httpd/global.conf

```diff
@@ -202,10 +202,8 @@ RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?collusion.*$ /$1lightbeam/ [L,R=301]
 RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/toolkit/download-to-your-devices(.*)$ /$1firefox/ [L,R=301]
 
 ## Home Page
-# Has to be this way because of old
```

Comment:
This comment reminds me. I think we have urls that are just 3 letters and would be matched by this. e.g. https://www.mozilla.org/ITU/. If this rule were at the bottom of global.conf it might be fine. Hmmm.